PrinterCommunication/PrinterConnection.cs

```diff
@@ -174,12 +170,8 @@ public bool WaitingForPositionRead
 
 		private double currentSdBytes = 0;
 
-		private PrinterMachineInstruction.MovementTypes extruderMode = PrinterMachineInstruction.MovementTypes.Absolute;
-
 		private double fanSpeed;
 
-		private bool firmwareUriGcodeSend = false;
-
```

Comment:
firmwareUriGcodeSend  is set to true when line start with "EXTENSIONS:" and contains "URI_GCODE_SEND" but is never read